src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age >= 0
+        && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+      {person.isMarried === true
+        && <p className="Person__partner">{`${person.partnerName} is my ${partner}`}</p>}
+
+      {person.isMarried === false
+        ? <p className="Person__partner">I am not married</p>
```

Comment:
you use it twice `<p className="Person__partner">`
for fix use [Conditional (ternary) operator](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Operators/Conditional_operator) like this:
```
<p className="Person__partner">
  {person.isMarried ? (
  
  ) : (
  
  )}
</p>
```